components/server/src/data/datamodel.json

```diff
@@ -4586,6 +4586,17 @@
                         "security_warnings"
                     ]
                 },
+                "landing_url": {
+                    "name": "URL to a Safety report in a human readable format",
+                    "short_name": "URL",
+                    "help": "If provided, users clicking the source URL will visit this URL instead of the Robot Framework report in XML format.",
```

Comment:
```suggestion
                    "help": "If provided, users clicking the source URL will visit this URL instead of the Safety report in JSON format.",
```